packages/components/src/index.html

```diff
@@ -36,8 +36,8 @@
 
       <!-- Language switch -->
       <post-language-menu
-        caption="Change the language"
-        description="The currently selected language is English."
+        text-change-language="Change the language"
+        text-current-language="The currently selected language is #name."
```

Comment:
The `#name` change has nothing to do with renaming the label properties and is not documented in a changeset. Either reset it or move it to a different PR.